src/sentry/static/sentry/app/data/forms/sentryApplication.jsx

```diff
@@ -51,6 +54,15 @@ const forms = [
         placeholder: 'e.g. https://example.com/sentry/setup/',
         help: 'The URL Sentry will redirect users to after installation.',
       },
+      {
+        name: 'verifyInstall',
+        label: 'Verify Installation',
+        type: 'boolean',
+        disabled: ({statusDisabled}) => statusDisabled,
```

Comment:
Where does `statusDisabled` come from?

Reply:
it's set [here](https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationDetails.jsx#L94)

Reply:
But how does it get passed to this anonymous function as an argument?

Reply:
oh. through the `additionalFieldProps` prop that eventually gets to https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/views/settings/components/forms/jsonForm.jsx#L228,